File: design-patterns/JavaDesignPatterns/src/com/bigboxcode/designpattern/adapter/transport/Bus.java
// Bus.java

package com.bigboxcode.designpattern.adapter.transport;

public class Bus implements Transport {
    @Override
    public int getNumberOfWheels() {
        return 4;
    }

    @Override
    public double getWeight() {
        return 10_000;
    }

    @Override
    public double getDistanceTravelled() {
        return 1_000;
    }

    @Override
    public double getTravelCostPerMile() {
        return 5;
    }
}

File: design-patterns/php/adapter/transportadapter/Bus.php
<?php
// Bus.php

namespace BigBoxCode\DesignPattern\Adapter\TransportAdapter;


class Bus implements Transport {
    public function getNumberOfWheels(): int {
        return 4;
    }

    public function getWeight(): float {
        return 10_000;
    }

    public function getDistanceTravelled(): float {
        return 1_000;
    }
    
    public function getTravelCostPerMile(): float {
        return 5;
    }
}